script.js

```diff
@@ -0,0 +1,104 @@
+const main = document.querySelector('.main')
+
+const cards = [
+    'images/0.jpg','images/1.jpg','images/2.jpg','images/3.jpg',
+    'images/4.jpg','images/5.jpg','images/6.jpg','images/7.jpg'
+]
+
+card = [...cards, ...cards]
+
+card.sort(() => Math.random() - 0.5)
+
+let userClick = []
+let check = []
+let scores = 0
+let gamePairs = card.length/2
+
+const createCard = () =>{
+    const game = document.createElement('div')
+    game.classList = "main__game"
+
+    main.appendChild(game)
+
+    card.forEach(item => {
```

Comment:
no. you can't create card and then for every card invoke this code. Create card should only create a single card.